settings.yml

```diff
@@ -1,7 +1,7 @@
 ---
 project: sln_voice
 title: XCORE-VOICE SOLUTION
-version: 2.1.0
+version: 2.2.0
```

Comment:
Are we sure that this is the only place that needs a version number update?  While I would hope that is the case, experience has taught me that many products and reference designs actually hard code the version number in multiple places.

Reply:
Yes - I recursively searched for 2.1.0 and 2.0.0 and there were no matches outside of changelog and modules.